libraries/ESP8266SSDP/ESP8266SSDP.cpp

```diff
@@ -155,12 +156,14 @@ SSDPClass::~SSDPClass() {
 
 bool SSDPClass::begin() {
   end();
-
+  
   _pending = false;
+  _st_is_uuid = false;
   if (strcmp(_uuid,"") == 0) {
 	uint32_t chipId = ESP.getChipId();
-	sprintf(_uuid, "38323636-4558-4dda-9188-cda0e6%02x%02x%02x",
-    (uint16_t) ((chipId >> 16) & 0xff),
+	sprintf(_uuid, "%s38323636-4558-4dda-9188-cda0e6%02x%02x%02x",
+  "uuid:",
```

Comment:
Why don't you include "uuid:" in the format string ?
(you can also put it in flash)
```
sprintf_P(_uuid, F("uuid:38323636-4558-4dda-9188-cda0e6%02x%02x%02x"),
    (uint16_t) ((chipId >> 16) & 0xff), ...
```

Reply:
I did not think of that actually 😃
Thanks for noticing I will change it now ..
Any other notes ?!

Reply:
Yes, same notice for `SSDPClass::setUUID` :)